pyaedt/filtersolutions_core/transmission_zeros.py

```diff
@@ -0,0 +1,258 @@
+            The default is blank.
+        position: str, optional
+            Position of the element creating transmission zero in the associated circuit.
+            The default is blank.
+        """
+        zero_bytes_value = bytes(zero, "ascii")
+        position_bytes_value = bytes(position, "ascii")
+        status = self._dll.updateTransmissionZerosTableRow(
+            row_index,
+            zero_bytes_value,
+            position_bytes_value,
+            self.table_format_to_bool(),
+        )
+        pyaedt.filtersolutions_core._dll_interface().raise_error(status)
+
+    def append_row(self, zero, position=""):
+        """Append ratios or frequencies at the end row of transmission zeros table.
+
+        Parameters
+        ----------
+        zero: str
+            Transmission zero ratio or frequency value.
+        position: str
+            Position of the element creating transmission zero in the associated circuit.
+        """
+        zero_bytes_value = bytes(zero, "ascii")
+        position_bytes_value = bytes(position, "ascii")
+        status = self._dll.appendTransmissionZerosTableRow(
+            zero_bytes_value, position_bytes_value, self.table_format_to_bool()
+        )
+        pyaedt.filtersolutions_core._dll_interface().raise_error(status)
+
+    def insert_row(self, row_index, zero, position=""):
+        """Insert ratios or frequencies at given index of the transmission zeros table.
+
+        Parameters
+        ----------
+        row_index: int
+            The row index on transmission zeros table. Starting value is ``0`` and maximum value is ``9``.
+        zero: str
+            Transmission zero ratio or frequency value.
+        position: str
+            Position of the element creating transmission zero in the associated circuit.
+        """
+        zero_bytes_value = bytes(zero, "ascii")
+        position_bytes_value = bytes(position, "ascii")
+        status = self._dll.insertTransmissionZerosTableRow(
+            row_index,
+            zero_bytes_value,
+            position_bytes_value,
+            self.table_format_to_bool(),
+        )
+        pyaedt.filtersolutions_core._dll_interface().raise_error(status)
+
+    def remove_row(self, row_index):
+        """Remove ratios or frequencies at given index of the transmission zeros table.
+
+        Parameters
+        ----------
+        row_index: int
+            The row index on transmission zeros table. Starting value is ``0`` and maximum value is ``9``.
+        """
+        status = self._dll.removeTransmissionZerosTableRow(row_index, self.table_format_to_bool())
+        pyaedt.filtersolutions_core._dll_interface().raise_error(status)
+
+    def clear_row(self):
+        """Clear all entries of the transmission zeros table."""
+        status = self._dll.clearTransmissionZerosTableRow(self.table_format_to_bool())
+        pyaedt.filtersolutions_core._dll_interface().raise_error(status)
+
+    def default_position(self):
```

Comment:
The name of this function sounds like it accesses the default position and returns a value. If it's perforning an action or setting a value (and not a property) we should make it a verb. How about `restore_default_position`?

Is it one position that changes or all of them in the table? If it's all of them, then it should be plural `restore_default_positions`.